adagrams/game.py

```diff
@@ -1,11 +1,111 @@
+
+import random
+import string
+
 def draw_letters():
-    pass
+    letter_pool = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    hand = []
+    random_letter = ""
+    while len(hand) < 10:
+        random_letter = random.choice(string.ascii_uppercase)
+        if letter_pool[random_letter] > 0:
+            letter_pool[random_letter] = letter_pool[random_letter] - 1
+            hand.append(random_letter)
+        
+    return hand
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    bank = []
+    word = word.upper()
+    for letter in letter_bank:
+        bank.append(letter)
```

Comment:
With this for loop, you are essentially copying over letter_bank into a new list. We have built in functions (.copy() and .deepcopy()) that'll do that for us!